src/StatusAlert/index.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './StatusAlert.scss';
+
+class StatusAlert extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.close = this.close.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.renderDialog = this.renderDialog.bind(this);
+
+    this.state = {
+      open: props.open,
+    };
+  }
+
+  componentDidMount() {
+    if (this.xButton) {
+      this.xButton.focus();
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.open !== this.props.open) {
+      this.setState({ open: nextProps.open });
+    }
+  }
+
+  componentDidUpdate(prevState) {
+    if (this.state.open && !prevState.open) {
+      this.xButton.focus();
+    }
+  }
+
+  close() {
+    this.setState({ open: false });
+    this.props.onClose();
+  }
+
+  handleKeyDown(e) {
+    if (e.key === 'Enter' || e.key === 'Escape') {
+      e.preventDefault();
+      this.close();
+    }
+  }
+
+  renderDialog() {
+    const { dialog } = this.props;
+
+    return (
+      <div className="alert-dialog">
+        { dialog }
+      </div>
+    );
+  }
+
+  renderDismissible() {
+    const { dismissible } = this.props;
+    let dismissButton = (<div />);
+    if (dismissible) {
+      dismissButton = (
+        <button
```

Comment:
Noting down our discussion -- the dismissible button formatting is generated from both the alert & close SASS files in order to render properly -- Will try to include the close partial into Button to see if it will continue to render correctly.